project.clj

```diff
@@ -23,22 +23,22 @@
                          :jvm-opts ^:replace ["-Xms1g" "-Xmx1g" "-server"]
                          :dependencies [[criterium "0.4.6"]
                                         [org.clojure/data.json "0.2.6"]
-                                        [clj-json "0.5.3"]]}}
+                                        [clj-json "0.5.3"]]}
+             :antq {:dependencies [[com.github.liquidz/antq "2.11.1276"]]}}
```

Comment:
Looking at the antq readme, it appears that it is supported as a plugin, is there a reason you chose to add it as a separate alias/command rather than using it in `:plugins` and having `lein antq` work?

Reply:
Good question! I did try antq as a lein plugin, but when I did that, it did not, for whatever reason, find outdated deps in GitHub Actions config. Expressing this way did the trick.